vendors/ti/boards/cc3220_launchpad/ports/secure_sockets/iot_secure_sockets.c

```diff
@@ -902,6 +902,207 @@ int32_t SOCKETS_Send( Socket_t xSocket,
 }
 /*-----------------------------------------------------------*/
 
+#if ( configPLATFORM_SOCKET_UDP_SUPPORT == 1 )
+    int32_t SOCKETS_Bind( Socket_t xSocket,
+                          const SocketsSockaddr_t * pxAddress,
+                          Socklen_t xAddressLength )
+    {
+        SlSockAddrIn_t xAddress;
+        SSocketContextPtr_t pxSocketContext;
+        _i16 sTIRetCode;
+        int32_t lRetCode = SOCKETS_ERROR_NONE;
+        uint32_t ulSocketNumber = ( uint32_t ) xSocket;
+
+        /* Remove compiler warnings about unused parameters. */
+        ( void ) xAddressLength;
+
+        /* Ensure that the socket and address is valid. */
+        if( ( prvIsValidSocket( ulSocketNumber ) == pdTRUE ) || ( pxAddress != NULL ) )
+        {
+            /* Shortcut for easy access. */
+            pxSocketContext = &( xSockets[ ulSocketNumber ] );
+
+            /* Connect. */
+            if( lRetCode == SOCKETS_ERROR_NONE )
+            {
+                /* Set up the address to connect to. */
+                xAddress.sin_family = SL_AF_INET;
+                xAddress.sin_port = pxAddress->usPort;
+                xAddress.sin_addr.s_addr = pxAddress->ulAddress;
+
+                /* Initiate connect. */
+                sTIRetCode = sl_Bind( pxSocketContext->sSocketDescriptor,
+                                      ( const SlSockAddr_t * ) &( xAddress ), /*lint !e9087 !e740 SlSockAddr_t behaves like a union of SlSockAddrIn_t.
+                                                                              * SlSockAddrIn_t is the appropriate interpretation of a IPv4 address. */
+                                      ( _i16 ) sizeof( SlSockAddrIn_t ) );
+
+                /* If connection is successful, mark the socket as connected. */
```

Comment:
nit: this comment describes the success case but the line beneath tests for the failure case.  I'd either reword the comment as _"If the connection wasn't successful, return a socket error"_ or delete the comment entirely.